jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/IPC.java

```diff
@@ -88,6 +88,11 @@ public class IPC {
 	 */
 	public static final boolean isZOS;
 
+	/**
+	 * True if operating system is Linux.
+	 */
+	public static final boolean isLinux;
+
```

Comment:
Please add this before `isWindows` to keep the `is*` fields in order.

Reply:
Updated